src/java.base/share/classes/java/lang/AbstractStringBuilder.java

```diff
@@ -179,11 +179,12 @@ int compareTo(AbstractStringBuilder another) {
         int count1 = this.count;
         int count2 = another.count;
 
+        byte coder = this.coder;
         if (coder == another.coder) {
-            return isLatin1() ? StringLatin1.compareTo(val1, val2, count1, count2)
+            return isLatin1(coder) ? StringLatin1.compareTo(val1, val2, count1, count2)
                               : StringUTF16.compareTo(val1, val2, count1, count2);
```

Comment:
```suggestion
            return isLatin1(coder) ? StringLatin1.compareTo(val1, val2, count1, count2)
                                                  : StringUTF16.compareTo(val1, val2, count1, count2);
```
As before, align the code